resolver_balancer_ext_test.go

```diff
@@ -57,7 +57,7 @@ func (s) TestResolverBalancerInteraction(t *testing.T) {
 	rb := manual.NewBuilderWithScheme(name)
 	rb.BuildCallback = func(_ resolver.Target, cc resolver.ClientConn, _ resolver.BuildOptions) {
 		sc := cc.ParseServiceConfig(`{"loadBalancingConfig": [{"` + name + `":{}}]}`)
-		cc.UpdateState(resolver.State{
+		rb.InitialState(resolver.State{
```

Comment:
Does this need to be changed? I think it may be unnecessary.

Reply:
used cc.UpdateState